### PR TITLE
Add code to show data and noise patterns to choose from in dash app

### PR DESCRIPTION
**Pull request by laurafroelich**

### CODE REVIEW

File: plots.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+def plot_normalize_data_pattern(data_function, x):
+    data = data_function(x)
+    data = normalize(data)
+    return tiny_plot(data)
+
+
+def normalize(data):
+    data = (data - np.min(data))
+    data = data/(np.max(data)/2)
```
Comment (rhaDHI) [inline plots.py:31]: When data is overwritten in line 30 you then take max of the overwritten data inline 31. Is that correct?

**Review by rhaDHI — COMMENTED** [review]
Please add a line in the readme about how to run the app.

File: plots.py (hunk 1 above)
Comment (laurafroelich) [reply plots.py:31]: Yes

**Comment by laurafroelich** [comment]
> 
> 
> Please add a line in the readme about how to run the app.

Good idea, I have added some instructions.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
 # Anomaly detection experimentation app
-App to explore anomaly detection for time series data.
\ No newline at end of file
+App to explore anomaly detection for time series data.
```
Comment (laurafroelich) [inline README.md:2]: Good idea, I have added instructions on running app.

**Review by rhaDHI — APPROVED** [review]

**Pull request merged**